src/main/scala/com/fullcontact/interview/RecordFinder.scala

```diff
@@ -1,7 +1,59 @@
+/*
+Author: FullContact
+Modified By: Jeff Willingham
+Creation Date: unknown
+Modification Date: 2020-07-28
+Purpose: This script inputs Queries.txt and Records.txt, performs transformations and writes outputs per specifications outlined in README.md.
+*/
+
 package com.fullcontact.interview
 
+import org.apache.spark.sql.{DataFrame, SaveMode, SparkSession}
+import org.apache.spark.sql.functions._
+
 object RecordFinder {
   def main(args: Array[String]): Unit = {
```

Comment:
Suppose we wanted to scale up the inputs to this job, to millions or billions of rows. Are there any performance issues you foresee in as we scale up? Are there any code changes you would want to make to support that scale up?